File: GuzzleProject/App/Model/Risk.php
<?php
	
	/**
	 * --------------------------------------------------------------------------
	 *   @Copyright : License MIT 2017
	 *
	 *   @Author : Alexandre Caillot
	 *   @WebSite : https://www.shiros.fr
	 *
	 *   @File : Risk.php
	 *   @Created_at : 10/03/2018
	 *   @Update_at : 10/03/2018
	 * --------------------------------------------------------------------------
	 */
	
	namespace App\Model;
	
	class Risk
	{
		/** @var int */
		protected $id;
		
		/** @var string */
		protected $name;
		
		/**
		 * @return int
		 */
		public function getId(): int
		{
			return $this->id;
		}
		
		/**
		 * @param int $id
		 */
		public function setId(int $id): void
		{
			$this->id = $id;
		}
		
		/**
		 * @return string
		 */
		public function getName(): string
		{
			return $this->name;
		}
		
		/**
		 * @param string $name
		 */
		public function setName(string $name): void
		{
			$this->name = $name;
		}
	}
	?>

File: HerokuProject/src/main/java/shiros/entity/Risk.java
package shiros.entity;

public class Risk {
	public static final String LOW = "Low";
	public static final String HIGH = "High";
	public static final int CEILING = 10000;
	
	protected int id;
	protected String name;
	
	public int getId() {
		return id;
	}
	public Risk setId(int id) {
		this.id = id;
		return this;
	}
	
	public String getName() {
		return name;
	}
	
	public Risk setName(String name) {
		this.name = name;
		return this;
	}
}
